styles.css

```diff
@@ -1,38 +1,624 @@
-body {
-  background: url("./images/background.jpg") no-repeat center center/cover;
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap");
+
+* {
+  scroll-behavior: smooth;
+  font-family: "Montserrat", sans-serif;
+  text-decoration: none;
+  margin: 0;
+  box-sizing: border-box;
 }
 
-.main-div {
-  height: 100%;
-  width: 100%;
-  position: relative;
+:root {
```

Comment:
colors are a bit dark. You can take suggestions from tailwind/ semantic ui/ bootstrap as Tanay sir mentioned